en.dict.email-reply.php

```diff
@@ -39,5 +39,9 @@
 	'UI-emry-select-attachments:Short' => 'Attachments',
 	'UI-emry-attachments-to-be-sent' => 'The following attachments will be sent:',
 	'UI-emry-select-attachments-tooltip' => 'Click to select the attachments to be sent',
+	'UI-emry-attachment-label-case-A' => '<b>%1$s</b> by %2$s at %3$s',
+	'UI-emry-attachment-label-case-B' => '<b>%1$s</b> by %2$s',
+	'UI-emry-attachment-label-case-C' => '<b>%1$s</b> at %3$s',
+	'UI-emry-attachment-label-case-D' => '<b>%1$s</b>',
```

Comment:
I also think the naming `case-A` to `case-D` were just examples, not to be actually used?